packages/ui/components/QuestionToggleTable/index.tsx

```
@@ -0,0 +1,75 @@
+  return (
+    <table className="mt-4 w-1/2 table-fixed">
+      <thead>
+        <tr className="text-left text-slate-800">
+          <th className="w-1/2 text-sm font-semibold">{capitalizeFirstLetter(type)} Fields</th>
+          <th className="w-1/4 text-sm font-semibold">Show</th>
+          <th className="w-1/4 text-sm font-semibold">Required</th>
+        </tr>
+      </thead>
+      <tbody>
+        {fields.map((field) => (
+          <tr className="text-slate-900">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)